hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -153,180 +192,128 @@ def collect_current_houdini_session(self, settings, parent_item):
-
-            # the item has been created. update the display name to
-            # include the node path to make it clear to the user how it
-            # was collected within the current session.
-            item.name = "%s (%s)" % (item.name, node.path())
-
-            if work_template:
-                item.properties["work_template"] = work_template
-
-            self._alembic_nodes_collected = True
-
-    def collect_tk_mantranodes(self, parent_item):
-        """
-        Checks for an installed `tk-houdini-mantranode` app. If installed, will
-        search for instances of the node in the current session and create an
-        item for each one with an output on disk.
-
-        :param parent_item: The item to parent new items to.
+        :returns: The appropriate publish name.
         """
-
         publisher = self.parent
-        engine = publisher.engine
 
-        mantranode_app = engine.apps.get("tk-houdini-mantranode")
-        if not mantranode_app:
-            self.logger.debug(
-                "The tk-houdini-mantranode app is not installed. "
-                "Will not attempt to collect those nodes."
-            )
-            return
-
-        try:
-            tk_mantra_nodes = mantranode_app.get_nodes()
-        except AttributeError, e:
-            self.logger.warning(
-                "Unable to query the session for tk-houdini-mantranode "
-                "instances. It looks like perhaps an older version of the "
-                "app is in use which does not support querying the nodes. "
-                "Consider updating the app to allow publishing their outputs."
-            )
-            return
-
-        # retrieve the work file template defined by the app. we'll set this
-        # on the collected alembicnode items for use during publishing.
-        work_template = mantranode_app.get_work_file_template()
-
-        for node in tk_mantra_nodes:
-
-            out_path = mantranode_app.get_output_path(node)
-
-            if not os.path.exists(out_path):
-                continue
-
-            self.logger.info(
-                "Processing sgtk_mantra node: %s" % (node.path(),))
-
-            # allow the base class to collect and create the item. it
-            # should know how to handle the output path
-            item = super(HoudiniSessionCollector, self)._collect_file(
-                parent_item,
-                out_path,
-                frame_sequence=True
-            )
-
-            # the item has been created. update the display name to
-            # include the node path to make it clear to the user how it
-            # was collected within the current session.
-            item.name = "%s (%s)" % (item.name, node.path())
-
-            if work_template:
-                item.properties["work_template"] = work_template
-
-            self._mantra_nodes_collected = True
+        logger = publisher.logger
+        logger.debug("Getting publish name for path: %s ..." % (path,))
```

Comment:
Let the logger substitute on-demand
```suggestion
        logger.debug("Getting publish name for path: %s ...", path)
```